respa_exchange/ews/notifications.py

```diff
@@ -27,42 +27,51 @@ class SubscribeRequest(EWSRequest):
     """
     version = 'Exchange2013'
 
-    def __init__(self, principal):
+    def __init__(self, principal, streaming=False):
```

Comment:
I don't see any place where this would be initialized with `streaming=False`. In the spirit of YAGNI, maybe it's worth it to just get rid of the `PullSubscriptionRequest` code here?

In any case I'd prefer `StreamingSubscribeRequest` and `PullSubscribeRequest` or similar, instead of an argument that makes the class behave rather entirely differently (as far as EWS is concerned)?